src/backend/resourcemanager/resourcemanager_RMSEG.c

```diff
@@ -151,6 +152,7 @@ int  initializeSocketServer_RMSEG(void)
 }
 #define SEGMENT_HEARTBEAT_INTERVAL (3LL * 1000000LL)
 #define SEGMENT_HOSTCHECK_INTERVAL (5LL * 1000000LL)
+#define SEGMENT_TMPDIRCHECK_INTERVAL (10 * 60LL * 1000000LL)
```

Comment:
using three gucs for these three parameters might be better. It will make future tuning and testing work much easier.